src/pyodide/internal/metadata.ts

```diff
@@ -58,6 +58,8 @@ const FORCE_NEW_VENDOR_PATH: boolean =
 export const IS_DEDICATED_SNAPSHOT_ENABLED: boolean =
   !!COMPATIBILITY_FLAGS.python_dedicated_snapshot;
 const EXTERNAL_SDK = !!COMPATIBILITY_FLAGS.enable_python_external_sdk;
+export const WORKFLOWS_IMPLICIT_DEPS =
+  !!COMPATIBILITY_FLAGS.python_workflows_implicit_dependencies;
```

Comment:
I think we should do `registerJsModule("_cloudflare_compat_flags", COMPATIBILITY_FLAGS)` somewhere and then we can get the compat flags with `from _cloudflare_compat_flags import WORKFLOWS_IMPLICIT_DEPS` or something like that.